cedar-policy-core/src/authorizer/partial_response.rs

```diff
@@ -317,17 +317,31 @@ impl PartialResponse {
         auth: &Authorizer,
```

Comment:
It seems like `reauthorize` (and all of `PartialResponse`) is not behind a feature flag, which would otherwise be convenient to require, to be able to rely on feature-flagged functions (in this case `with_unknowns_mapper` if feature-gated) downstream